src/traits.rs

```diff
@@ -590,29 +529,35 @@ pub trait SuffStat<X> {
 /// };
 ///
 /// // Using cache improves runtime
-/// assert!(t_no_cache.as_nanos() >  t_cache.as_nanos());
+/// assert!(t_no_cache.as_nanos() > t_cache.as_nanos());
 /// ```
-pub trait ConjugatePrior<X, Fx>: Rv<Fx>
+pub trait ConjugatePrior<X, Fx>: Sampleable<Fx>
```

Comment:
is there a reason `Rv<Fx>` was replaced with `Sampleable<Fx>` in the trait bounds?

Reply:
`StickBreaking` can't implement `HasDensity<StickBreakingDiscrete>`, because `StickBreakingDiscrete` has an infinite parameter space.

Because of that, we can't have
```rust
impl ConjugatePrior<usize, StickBreakingDiscrete> for StickBreaking
```

Reply:
The density evaluation wasn't used in the `rv` crate. If Lace needs it, maybe we can add a bound for the density space? In this case we have
```rust
impl HasDensity<PartialWeights> for StickBreaking
```